rpc/server.go

```diff
@@ -91,6 +93,13 @@ func (s *Server) SetHTTPBodyLimit(limit int) {
 	s.httpBodyLimit = limit
 }
 
+// SetReadLimits sets the limit for max message size to limit the request message size.
+//
+// This method would be useful to prevent the server from OOM when taking too large messages.
+func (s *Server) SetReadLimits(maxMessageSize int64) {
```

Comment:
sorry I'm stupid where is this called?

Reply:
yeah, will be called in Sei-chain once this is merged